client/components/Navbar.js

```diff
@@ -25,10 +32,15 @@ const Navbar = ({ handleClick, isLoggedIn }) => (
           {/* The navbar will show these links before you log in */}
           <Link to="/login">Login</Link>
           <Link to="/signup">Sign Up</Link>
-          <Link to="/cart">
-            Cart
-            {/*<ShoppingCartRoundedIcon />*/}
-          </Link>
+          <Link to="/products">Products</Link>
+          {cart.length === 0 ? (
+            <Link to="/cart">Cart{/*<ShoppingCartRoundedIcon />*/}</Link>
+          ) : (
+            <Link to="/cart">
+              Cart ({cart.reduce((a, c) => a + c.count, 0)})
```

Comment:
this formula calculates the total number of items in the cart and adds the number to the Navbar